### ISSUE TITLE
Template URDF

### ISSUE THREAD

**Issue by Guillaumebeuzeboc** [ros-industrial/ros_qtc_plugin#61]
Hello,

I noticed that when I create a urdf file from the plugin the closing tag seems wrong.
auto-generated template:

```
<?xml version="1.0" ?>
<robot name="tmp" xmlns:xacro="http://ros.org/wiki/xacro">
    <link name="base"/>
    <link name="tool0"/>

</launch>
```

The closing tag is supposed to be  `</robot>` and not `</launch>`

Not a big issue, this could be included in another PR.

**Comment by Levi-Armstrong** [comment]
Thank you for the feed back. I will get it fixed in an up coming PR unless someone else get to it first.

**Comment by Levi-Armstrong** [comment]
Fixed by PR #66
